From 77a5cac3d7fbd1525da42e560d3ba45de6f83cb6 Mon Sep 17 00:00:00 2001
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Wed, 3 Feb 2021 00:53:55 +0000
Subject: [PATCH] gnu: log4cl: Update to 1.1.3-1.8c48d6f

* gnu/packages/lisp-xyz.scm
  [native-inputs] add stefil to enable tests
  [version] set version style alike majority of non versioned CL packages
  Gentle refactor of the package structure
---
 gnu/packages/lisp-xyz.scm | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/gnu/packages/lisp-xyz.scm b/gnu/packages/lisp-xyz.scm
index 736dbfca37..f8079f457a 100644
--- a/gnu/packages/lisp-xyz.scm
+++ b/gnu/packages/lisp-xyz.scm
@@ -4669,12 +4669,11 @@ theory accurate to internal-time-units-per-second.")
   (sbcl-package->ecl-package sbcl-cl-log))
 
 (define-public sbcl-log4cl
-  (let ((commit "611e094458504b938d49de904eab141285328c7c")
+  (let ((commit "8c48d6f41d3a1475d0a91eed0638b9eecc398e35")
         (revision "1"))
     (package
       (name "sbcl-log4cl")
-      (build-system asdf-build-system/sbcl)
-      (version "1.1.2")
+      (version (git-version "1.1.3" revision commit))
       (source
        (origin
          (method git-fetch)
@@ -4683,16 +4682,17 @@ theory accurate to internal-time-units-per-second.")
                (commit commit)))
          (file-name (git-file-name name version))
          (sha256
-          (base32
-           "08jly0s0g26b56hhpfizxsb4j0yvbh946sd205gr42dkzv8l7dsc"))))
-      ;; FIXME: tests require stefil, sbcl-hu.dwim.stefil wont work
-      (arguments
-       `(#:tests? #f))
-      (inputs `(("bordeaux-threads" ,sbcl-bordeaux-threads)))
-      (synopsis "Common Lisp logging framework, modeled after Log4J")
+          (base32 "0166d9aip366pbpdk5gsi2f6xad6q61lssxgbrypa8zslwjn8736"))))
+      (build-system asdf-build-system/sbcl)
+      (native-inputs
+       `(("stefil" ,sbcl-stefil)))
+      (inputs
+       `(("bordeaux-threads" ,sbcl-bordeaux-threads)))
       (home-page "https://github.com/7max/log4cl")
-      (description "This is a Common Lisp logging framework that can log at
-various levels and mix text with expressions.")
+      (synopsis "Common Lisp logging framework, modeled after Log4J")
+      (description
+       "This is a Common Lisp logging framework that can log at various levels
+and mix text with expressions.")
       (license license:asl2.0))))
 
 (define-public cl-log4cl
-- 
2.30.0

